api/app/Http/Controllers/Api/UserController.php

```diff
@@ -52,4 +58,19 @@ public function delete(): JsonResponse
 
         return $this->emptyResponse();
     }
+
+    public function updatePassword(UserPasswordRequest $request): JsonResponse
+    {
+        try {
+            $this->updateUserPasswordAction->execute(new UpdateUserPasswordRequest(
+                Auth::id(),
+                $request->get('oldPassword'),
+                $request->get('password'),
+            ));
+
+            return $this->emptyResponse();
+        } catch (\Exception $exception) {
+            return $this->errorResponse($exception->getMessage(), $exception->getCode());
+        }
```

Comment:
You do not need to handle the exception manually, it is already done in the `Handler`. Also, the exception code is not always the HTTP code.